composer.json

```diff
@@ -106,7 +106,8 @@
         "patches": {
             "drupal/core": {
                 "#3444246: Create a config action to set the status of the entity": "./patches/core/recipes-103.patch",
-                "Add config actions to make a bundle of an entity type translatable": "./patches/core/make-translatable-action.patch"
+                "Add config actions to make a bundle of an entity type translatable": "./patches/core/make-translatable-action.patch",
+                "Give block content entities bundle-level permissions granularity": "./patches/core/block-content-granularity.patch"
```

Comment:
Should patches like this reference a drupal.org issue?

Reply:
Yes, but this one has no d.o issue. I just spun up the patch quickly.

If there _is_ an existing issue with a patch (or you want to create one), let's use that.

Reply:
Is there a core issue for this, feels like an oversight given we do have bundle perms for them since drupal.org/node/1975064

Reply:
That is good. So if there is no #123456 in the patch description, then it is owned by the prototype.

Reply:
More or less, yeah. The rules are loose; patches are completely internal, so we can name them however we want.